lib/devise/secure_password/controllers/active_helpers.rb

```diff
@@ -27,7 +27,8 @@ def skip_current_controller?
             'Devise::PasswordsWithPolicyController#edit',
             'Devise::PasswordsWithPolicyController#update',
             'DeviseInvitable::RegistrationsController#edit',
-            'DeviseInvitable::RegistrationsController#update'
+            'DeviseInvitable::RegistrationsController#update',
+            'Devise::DeviseAuthyController#POST_verify_authy'
```

Comment:
We will want to rework this section as registering skipped controllers will not be maintainable, especially in a flexible framework like devise.  We can sidestep this issue by following other devise conventions of exposing controller checks instead of it's own before_action and then determining skipping behavior.